frb_rust/src/ffi/mod.rs

```diff
@@ -27,3 +27,12 @@ pub type Channel = allo_isolate::Isolate;
 pub mod io;
 #[cfg(not(wasm))]
 pub use io::*;
+
+#[cfg(feature = "chrono")]
+#[inline]
+pub fn wire2api_timestamp(ts: i64) -> (i64, u32) {
```

Comment:
Would be great to use `struct Whatever {somename: i64, anothername: u32}`, o/w may be confusing what is that i64 and u32

Reply:
resolved in af8b2ac35fdb7f354048b1a7e6b74df086083eec